modules/mvt/src/lib/binary-vector-tile/features-to-binary.js

```diff
@@ -252,6 +255,26 @@ function handlePolygon(geometry, polygons, indexMap, coordLength, properties) {
       polygons.primitivePolygonIndices[indexMap.polygonRing++] = indexMap.polygonPosition;
       indexMap.polygonPosition += (end - start) / coordLength;
     }
+
+    // Triangulate polygon using earcut
```

Comment:
This is a long function - seems like the below should go into its own helper function (perhaps in `@loaders.gl/math`

Reply:
Done